source/index.txt

```diff
@@ -0,0 +1,74 @@
+===================
+MongoDB Java Driver
+===================
+
+.. default-domain:: mongodb
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /quick-start
+   /usage-examples
+   /fundamentals
+   API Documentation <http://mongodb.github.io/mongo-java-driver/4.0/apidocs/mongodb-driver-sync/index.html>
+   /issues-and-help
+   /compatibility
+
+Introduction
+------------
+
+The official MongoDB Java Driver enables Java applications to connect to
+MongoDB and work with data.
+
+If you are working with asynchronous stream processing and reactive
+streams, we recommend the Reactive Streams Driver.
```

Comment:
Should we include a blank link here so it's not forgotten that this should link to something?

Reply:
I guess I'll just link to the current Reactive Streams driver reference docs since we're migrating one at a time.